internal/consensus/reactor_test.go

```diff
@@ -287,7 +289,7 @@ func TestReactorWithEvidence(t *testing.T) {
 	n := 4
 	testName := "consensus_reactor_test"
 	tickerFunc := newMockTickerFunc(true)
-	appFunc := newCounter
+	appFunc := newPersistentKVStore
```

Comment:
If it's just a test does have to be the persistentKVStore or can we use just KVStore

Reply:
it doesn't, I can add a func for non-persistent, but in short scenarios it is similar or do you see the writing to a db as a problem?

Reply:
I was suggesting it because I thought it might be faster to not use a db but if there's almost no difference then you can leave it

Reply:
i can make the change, its an easy change